Automattic/newspack-plugin
# Popups Wizard

### All Submissions:


### Changes:

This PR adds a new Wizard, dedicated to the Popups. Designed is based on pamTN9-1bQ-p2

### How to test?

Test with `newspack-popups` on https://github.com/Automattic/newspack-popups/pull/112.

1. Switch to this branch
2. WP sidebar, do you see "Popups" under Newspack?
3. Navigate to Newspack > Dashboard. Do you see a "Popup" card?
4. Go to Newspack > Popups
5. Have fun testing all functionalities

### Other information:
